.circleci/config.yml

```diff
@@ -1,61 +1,33 @@
 version: 2.1
 
-orbs:
-
-  # docker vulnerability scanning with AWS ECR
-  # reference this is the orb usage guide for aws-ecr:
-  # https://circleci.com/orbs/registry/orb/circleci/aws-ecr
-  # required environment variables:
-  # https://circleci.com/docs/2.0/ecs-ecr/#configure-circleci-environment-variables
-  aws-ecr: circleci/aws-ecr@6.15.3
-
 workflows:
-  version: 2
   build-and-test:
     jobs:
       - docker_image:
-          context: docker-hub
+          context:
+            - docker-hub
+            - aws-ecr
       - test
 
-      # use aws-ecr CircleCI Orb as a docker image vulnerability scanner.
-      - aws-ecr/build-and-push-image:
-          name: docker_image_and_vuln_scanning_with_ecr
-          # docker vulnerability scanning with AWS ECR!
-          # references:
-          #
-          #   https://circleci.com/orbs/registry/orb/circleci/aws-ecr
-          #   https://circleci.com/docs/2.0/ecs-ecr/
-          #
-          # The AWS ECR repo name, example: remind101/acme-inc
-          repo: "$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME"
-          # tags for this image, comma separated.
-          tag: "latest,$CIRCLE_BRANCH,$CIRCLE_SHA1"
-          # create the AWS ECR repo if it does not exist already.
-          create-repo: true
-          # provide the aws-ecr context (environment variables) to push docker_image into ecr.
-          # https://circleci.com/docs/2.0/ecs-ecr/#configure-circleci-environment-variables
-          context: aws-ecr
-
 jobs:
   # docker-build is a small script for building, tagging and pushing docker images within CircleCI.
   docker_image:
     docker:
       # build docker images and push them into a remote docker hub.
-      # https://github.com/remind101/docker-build/commits/5af523d5343bb852aa41c56b47dd470b9d3beccd
-      - image: remind101/docker-build@sha256:d8087702ebd201855dd98dff98bb122a2cb18149edb1e1d6ee3b09e635b0f43b
+      - image: remind101/docker-build@sha256:b035f1b056c4186ebda5faa5979d2f9c4c3a14ed9ff8b5b5b1e0f125a9afea0d
```

Comment:
Can you leave in/update the comment with the git sha? It's nice to easily get back to that.